### ISSUE TITLE
Stats (Today, Scheduled, Recording, …) do not meet minimum color contrast ratio

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#534]
This is the same problem as in #532 

The text in the Stats container does not meet the minimum color contrast ratio.

Currently the ratio is about 2:1 but should be at least 4.5:1.

This has a serious impact on people with low vision.
